docs/http.rst

```diff
@@ -92,6 +92,9 @@ Unless otherwise specified, all calls return the following error codes:
    - errno 110 - Invalid crypto keys specified
    - errno 111 - Missing Required Header
 
+       - Missing TTL Header - Include the Time To Live header (`ref<https://tools.ietf.org/html/draft-ietf-webpush-protocol#section-6.2>`_)
+       - Missing Crypto Headers - Include the appropriate encryption headers (`ref<https://webpush-wg.github.io/webpush-encryption/#rfc.section.3.2>`_ and `ref<https://tools.ietf.org/id/draft-thomson-webpush-vapid-02.html#rfc.section.4>`_)
```

Comment:
Yes. i figured "ref" (being a fairly old dictionary standard to indicate reference) was better and shorter than "See specification declaration here") If there is a different standard of reference declaration, I'm fine using that.